api/Tests/ApiTests/ApiTestCase.swift

```diff
@@ -9,12 +9,12 @@ import XSlack
 
 @testable import Api
 
-class ApiTestCase: XCTestCase {
+class ApiTestCase: XCTestCase, @unchecked Sendable {
```

Comment:
unsafe, and theoretically racy, but i don't care, because this is only for unit tests. i'm very sensitive to tests being flaky, which ours aren't, one day when we update to swift testing, we can probably drop some of this baggage, but it's not worth a refactor now, was my thought.